Gopkg.toml

```diff
@@ -65,10 +42,6 @@
   name = "gopkg.in/src-d/go-git.v4"
   version = "4.1.0"
 
-[[constraint]]
-  name = "github.com/spf13/pflag"
```

Comment:
Pflag is needed. All Unix programs must be posix compatible.

Reply:
is this an agreed constraint across the company?

Reply:
I don't think so. We definitely have an agreement to use cobra for bigger projects; this implicitly states pflag.